packages/picasso.js/src/core/chart/index.js

```diff
@@ -426,6 +426,12 @@ function chartFn(definition, context) {
     // Let the "components" array determine order of components
     currentComponents = components.map((comp) => {
       const idx = findComponentIndexByKey(comp.key);
+
+      // Component should not be updated
+      if (excludeFromUpdate.length && excludeFromUpdate.indexOf(comp.key) > -1) {
```

Comment:
I would skip the length check. It's an array or it should throw. Similar to how we assume the `components` setting is an array.

Reply:
Done!